src/locale/sr-cyrl.js

```diff
@@ -9,32 +9,42 @@ var translator = {
     words: {
         //Different grammatical cases
         ss: ['секунда', 'секунде', 'секунди'],
-        m: ['један минут', 'једне минуте'],
+        m: ['један минут', 'једног минута'],
         mm: ['минут', 'минуте', 'минута'],
         h: ['један сат', 'једног сата'],
         hh: ['сат', 'сата', 'сати'],
+        d: ['један дан', 'једног дана'],
         dd: ['дан', 'дана', 'дана'],
+        M: ['један месец', 'једног месеца'],
         MM: ['месец', 'месеца', 'месеци'],
+        y: ['једну годину', 'једне године'],
         yy: ['година', 'године', 'година'],
     },
     correctGrammaticalCase: function (number, wordKey) {
-        return number === 1
-            ? wordKey[0]
-            : number >= 2 && number <= 4
-            ? wordKey[1]
-            : wordKey[2];
+        if (number % 10 >= 1 && number % 10 <= 4 && (number % 100 < 10 || number % 100 >= 20)) {
+            return number % 10 === 1 ? wordKey[0] : wordKey[1]
+        }
+        return wordKey[2]
     },
-    translate: function (number, withoutSuffix, key) {
+    translate: function (number, withoutSuffix, key, isFuture) {
         var wordKey = translator.words[key];
+
         if (key.length === 1) {
-            return withoutSuffix ? wordKey[0] : wordKey[1];
-        } else {
-            return (
-                number +
-                ' ' +
-                translator.correctGrammaticalCase(number, wordKey)
-            );
+            // Nominativ
+            if (key === 'y' && withoutSuffix) return 'једна година'
+            return isFuture || withoutSuffix ? wordKey[0] : wordKey[1];
         }
+
+        const word = translator.correctGrammaticalCase(number, wordKey)
+        // Nominativ
+        if (
+            key === 'yy' &&
+            withoutSuffix &&
+            word === 'godinu') {
```

Comment:
I'm not sure "godinu" is correct here?

Reply:
"godinu" is used in the first grammatical case when using suffix (pre jednu godinu, pre 21 godinu, pre 31 godinu). However if the suffix is not used (thus checking for `withoutSuffix`) Nominativ is used (jedna godina, 21 godina, 31 godina). By checking for "godinu" I'm checking for the first grammatical case, and if it's the first grammatical case in use, "godinu" is changed to "godina".

Reply:
I see now what you meant 😅. No, "godinu" is not correct in this case.